### PR TITLE
Using allowBrokenSymLinks findOptions while calling findMatch API.

### PR DESCRIPTION
**Pull request by navin22**
- Removing the patchwork that was done earlier to unblock the customers.
- using vsts-task-lib 2.5.0
- 2.5.0 doesn't support sending telemetry, using the alternative agent code to publish telemetry.

### CODE REVIEW

File: Tasks/PublishTestResultsV2/publishtestresults.ts [hunk 1]
```diff
@@ -41,20 +41,13 @@ async function run() {
...
-            // Will remove this once we have right api in vsts-task-lib.
-            const findOptions = <tl.FindOptions>{
-                followSpecifiedSymbolicLink: false,
-                followSymbolicLinks: false
-            };
-            matchingTestResultsFiles = tl.findMatch(searchFolder, testResultsFiles, findOptions);
-        }
+        // Sending allowBrokenSymbolicLinks as true, so we don't want to throw error when symlinks are broken.
+        // And can continue with other files if there are any.
+        const findOptions = <tl.FindOptions>{
```
Comment (ericsciple) [inline Tasks/PublishTestResultsV2/publishtestresults.ts]: do you want to follow the rest of the defaults for a consistent experience across tasks?

that is, wouldn't these options make more sense:
```
        allowBrokenSymbolicLinks: true,
        followSpecifiedSymbolicLink: true,
        followSymbolicLinks: true
```

Comment (navin22) [reply Tasks/PublishTestResultsV2/publishtestresults.ts]: Okay, they are by default true right and allow broke symbolic links was false. ?

Comment (ericsciple) [reply Tasks/PublishTestResultsV2/publishtestresults.ts]: yes when findOptions is undefined, those are the defaults that are applied.

in this case you are providing the findOptions.

Comment (navin22) [reply Tasks/PublishTestResultsV2/publishtestresults.ts]: Okay, Fixed it :)

**Pull request merged**